R/run.R

```diff
@@ -144,6 +189,11 @@ CmdStanRun <- R6::R6Class(
           self$args$compose_all_args(
             idx = j,
             output_file = private$output_files_[j],
+            profile_file = if (!is.null(private$profile_files_)) {
+              private$profile_files_[j]
+            } else {
+              NULL
+            },
```

Comment:
Do we need the `if else` here or can we just do 

```r
profile_file = private$profile_files_[j]
```

like we do for `latent_dynamics_file` in the next line?

Reply:
No we dont need it. Thanks!